frontend/src/hooks/api/actions/useApiTokensApi/useApiTokensApi.ts

```diff
@@ -12,8 +13,12 @@ const useApiTokensApi = () => {
         propagateErrors: true,
     });
 
-    const deleteToken = async (secret: string) => {
-        const path = `api/admin/api-tokens/${secret}`;
+    const deleteToken = async (secret: string, project?: string) => {
+        const path = formatApiPath(
+            `api/admin/api-tokens/${secret}${
```

Comment:
This should be **api/admin/project/:projectId/api-tokens/:secret** rather.

Reply:
Also, since you dont have /project word inside the path, are you sure it works currently?

Reply:
Yes, that would require a different Controller in the BE - which is an overkill for what we are doing imo.  The endpoint sitting under api-token controlleris `admin/api-tokens/${secret}/{project}` mostly serves to solve the double permission problem

Reply:
Yes it works, the rbac middleware check for `product` or `productId` in the request params
![Screenshot 2023-02-09 at 16 50 24](https://user-images.githubusercontent.com/104830839/217846609-cfed8514-378e-456c-8850-ce9813566367.png)